main.go

```diff
@@ -101,7 +101,17 @@ func main() {
 		os.Exit(2)
 		return
 	}
-
+	if *enableReap {
+		if os.Getpid() == 1 {
+			// Clean up zombie processes caused by incorrect crontab commands
+			// Use forkExec to avoid random waitid errors
+			// https://github.com/aptible/supercronic/issues/88
+			// https://github.com/aptible/supercronic/issues/171
+			logrus.Info("will cleaning up zombie processes")
+			forkExec()
+			return
+		}
+	}
```

Comment:
Update for opt-out and log if the process is configured to reap but could not because it isn't PID 1.

```suggestion
	if !*disableReap {
		if os.Getpid() == 1 {
			// Clean up zombie processes caused by incorrect crontab commands
			// Use forkExec to avoid random waitid errors
			// https://github.com/aptible/supercronic/issues/88
			// https://github.com/aptible/supercronic/issues/171
			logrus.Info("reaping dead processes")
			forkExec()
			return
		}
		
		logrus.Warn("process reaping disabled, not pid 1")
	}
```